crypto/property/property.c

```diff
@@ -358,7 +362,7 @@ int ossl_method_store_fetch(OSSL_METHOD_STORE *store, int nid,
     else
         ret = 0;
     ossl_property_unlock(store);
-    ossl_property_free(pq);
+    ossl_property_free(p2);
```

Comment:
Make sure to initialise `p2`

Reply:
`p2` is initialised to NULL above.

I welcome suggestions and would love a patch commit to correct my mis-comprehensions.

The bottom line is that I understand the property code well enough but I do not understand the higher levels.

Reply:
I.e. add a patch here or create a new PR.

Reply:
D'oh, I'm blind...